keps/sig-testing/3041-node-conformance-and-features/README.md

```diff
@@ -0,0 +1,759 @@
+- Remove the `[NodeSpecialFeature:]` label.
+
+### Non-Goals
+
+- As `[NodeConformance]` may be a confusing term and may be confused with the
+  k8s Conformance tests, renaming of it is not in a scope of this KEP.
+
+## Proposal
+
+### Clarify the semantic of a `[Feature:]` label
+
+The current [feature definition is](https://github.com/kubernetes/community/blob/32a1c14d04ff78684d78b827ac7c49f70352d509/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests):
+
+> * `[Feature:.+]`: If a test has non-default requirements to run or targets
+>   some non-core functionality, and thus should not be run as part of the
+>   standard suite, it receives a `[Feature:.+]` label, e.g.
+>   `[Feature:Performance]` or `[Feature:Ingress]`. `[Feature:.+]` tests are not
+>   run in our core suites, instead running in custom suites. If a feature is
+>   experimental or alpha and is not enabled by default due to being incomplete
+>   or potentially subject to breaking changes, it does not block PR merges, and
+>   thus should run in some separate test suites owned by the feature owner(s)
+>   (see Continuous Integration below).
+
+Feature will indicate that things may NOT work on all k8s distros and/or node
+capabilities. This flag will not indicate the Feature Gate state.
+
+Proposed change:
+
+> * `[Feature:.+]`: If a test validates functionality that may only work when
+>   specific node capabilities are enabled or when test depends on the
+>   functionality of underlying components like container runtime, and thus may
+>   need to be skipped on certain environments, it receives a `[Feature:.+]`
+>   label, e.g. `[Feature:Swap]`. This label can also be applied to signify that
+>   test validates non-core functionality, like `[Feature:Ingress]`.
+>   `[Feature:.+]` tests are not run in a core suites and run separately.
+>   `[Feature:+]` label should not be confused with `[FeatureGate:]` label.
+
+And another one:
+
+> * `[FeatureGate:.+]`: If a test only works when the certain feature gate is
+>   enabled it receives a `[FeatureGate:.+]` label. `[FeatureGate:.+]` tests
+>   must also be marked with the status of this feature gate: `[Alpha]`,
+>   `[Beta]`, `[Stable]`, `[Deprecated]`. This label helps to skip tests that
+>   should not work on specific k8s distributive that has a certain feature gate
+>   disabled. This label has to be removed when feature gate value is locked or
+>   feature gate is removed.
+
+### Feature label clean up
+
+It is already true today that only a handful of tests marked as `Feature` don't
+have `Serial` or `Disruptive` labels as well. Only these tests has `Feature`
+label without `Serial` or `Disruptive` and most of them just degraded,
+and don’t actually need the `Feature` label any longer. Going forward, `Features`
+that require a special environment outside of predefined `LinuxOnly`, `Serial`, etc.
+will need to define their "custom" labels and be commented with the description
```

Comment:
By "custom", are you referring to `[Feature:]` tag or an entirely new tag? If it's the first case, I think some clarification might help.

Reply:
entirely new tag